Assignment 2/180501105/robberThief.py

```diff
@@ -0,0 +1,9 @@
+s=int(input("Hello robber! How much did you rob?"))
+if s<5000:
+    b="Loss"
+elif (s>5000 and s<10000):
+    b="Profit is Moderate"
+elif s>20000:
+    b="Profit is Good"
+print(b)
+
```

Comment:
Make sure your code covers all equal to conditions also

Reply:
changes made